frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -51,9 +51,9 @@ const cancelUpdate = (cv: ClusterVersionKind) => {
   });
 };
 
-const clusterAutoscalerReference = referenceForModel(ClusterAutoscalerModel);
+export const clusterAutoscalerReference = referenceForModel(ClusterAutoscalerModel);
```

Comment:
I don't think this needs to be exported since it's not referenced from the tests